docs/cloud-runner/01-introduction.md

```diff
@@ -0,0 +1,70 @@
+# Introduction
+## Concept - What Does Cloud Runner Do
+
+**Cloud Runner enables you to send your project to run workflows on a Cloud Provider such as AWS or GCP, most commonly to build and test your project.** 
```

Comment:
```suggestion
**Cloud Runner enables you to run build and test workflows in the cloud, right from GitHub actions. Builder will automatically provision an environment at a Cloud Provider such as GCP and AWS. It will then send the project to be built and/or tested depending on your workflow configuration.** 
```